static/app/views/explore/charts/index.tsx

```diff
@@ -1,5 +1,126 @@
-interface ExploreChartsProps {}
+import styled from '@emotion/styled';
 
-export function ExploreCharts({}: ExploreChartsProps) {
-  return <div>TODO: visualize charts</div>;
+import {getInterval} from 'sentry/components/charts/utils';
+import {CompactSelect} from 'sentry/components/compactSelect';
+import {CHART_PALETTE} from 'sentry/constants/chartPalette';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {tooltipFormatter} from 'sentry/utils/discover/charts';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import Chart, {ChartType} from 'sentry/views/insights/common/components/chart';
+import ChartPanel from 'sentry/views/insights/common/components/chartPanel';
+import {useSpanIndexedSeries} from 'sentry/views/insights/common/queries/useDiscoverSeries';
+import {CHART_HEIGHT} from 'sentry/views/insights/database/settings';
+
+import {useChartInterval} from '../hooks/useChartInterval';
+import {useChartType} from '../hooks/useChartType';
+import {useVisualize} from '../hooks/useVisualize';
+
+interface ExploreChartsProps {
+  query: string;
 }
+
+const exploreChartTypeOptions = [
+  {
+    value: ChartType.LINE,
+    label: t('Line'),
+  },
+  {
+    value: ChartType.AREA,
+    label: t('Area'),
+  },
+  {
+    value: ChartType.BAR,
+    label: t('Bar'),
+  },
+];
+
+// TODO: Update to support aggregate mode and multiple queries / visualizations
+export function ExploreCharts({query}: ExploreChartsProps) {
+  const pageFilters = usePageFilters();
+  const [visualize] = useVisualize();
+  const [chartType, setChartType] = useChartType();
+  const [interval, setInterval, intervalOptions] = useChartInterval();
+
+  const series = useSpanIndexedSeries(
+    {
+      search: new MutableSearch(query ?? ''),
+      yAxis: [visualize],
+      interval: interval ?? getInterval(pageFilters.selection.datetime, 'metrics'),
```

Comment:
Is this using the right signature for `getInterval`? It takes a fidelity as a second parameter but you're passing `'metrics'`

Reply:
Heyo @Zylphrex, I took this default from the [tracesChart.tsx](https://github.com/getsentry/sentry/blob/master/static/app/views/traces/tracesChart.tsx#L155). "metrics" is a valid option.